backend/infrahub/groups/tasks.py

```diff
@@ -5,14 +5,20 @@
 from infrahub.core.constants import InfrahubKind
 from infrahub.groups.models import RequestGraphQLQueryGroupUpdate
 from infrahub.services import services
-from infrahub.workflows.utils import add_branch_tag
+from infrahub.workflows.utils import add_tags
 
 
-@flow(name="update_graphql_query_group", flow_run_name="Update GraphQLQuery Group {model.query_name}")
+@flow(name="graphql-query-group-update", flow_run_name="Update GraphQLQuery Group '{model.query_name}'")
 async def update_graphql_query_group(model: RequestGraphQLQueryGroupUpdate) -> None:
     """Create or Update a GraphQLQueryGroup."""
 
-    await add_branch_tag(branch_name=model.branch)
+    # If there is only one subscriber, associate the task to it
+    # If there are more than one, for now we can't associate all of them
+    related_nodes = []
+    if model.subscribers and len(model.subscribers) == 1:
```

Comment:
```python
class RequestGraphQLQueryGroupUpdate(BaseModel):
    """Sent to create or update a GraphQLQueryGroup associated with a given GraphQLQuery."""

    branch: str = Field(..., description="The branch to target")
    query_name: str = Field(..., description="The name of the GraphQLQuery that should be associated with the group")
    query_id: str = Field(..., description="The ID of the GraphQLQuery that should be associated with the group")
    related_node_ids: list[str] = Field(..., description="List of nodes related to the GraphQLQuery")
    subscribers: list[str] = Field(..., description="List of subscribers to add to the group")
    params: dict[str, str] = Field(..., description="Params sent with the query")
```

The model is defined so that subscribers would be expected to be a list. So it would be enough with `len(model.subscribers) == 1` here.